ACS_child_allowance_rough_draft.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+import pandas as pd
+import microdf as mdf
+
+raw = pd.read_stata(
```

Comment:
```suggestion
person = pd.read_stata(
```
The extra `raw` piece is helpful for notebooks to avoid reloading large files, but if it's a Python script that runs all together it's not necessary.